<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>ファイル(&amp;F)</string>
    </property>
    <addaction name="action_C"/>
    <addaction name="action_Q"/>
   </widget>
   <widget class="QMenu" name="menu_C">
    <property name="title">
     <string>フラーレン指定(&amp;C)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_M">
    <property name="title">
     <string>モード(&amp;M)</string>
    </property>
    <addaction name="action_Z"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>入出力(&amp;O)</string>
    </property>
    <addaction name="action_M"/>
    <addaction name="action_R"/>
    <addaction name="action_6"/>
    <addaction name="action_S"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>ヘルプ(&amp;H)</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="action_A"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_C"/>
   <addaction name="menu_M"/>
   <addaction name="menu_O"/>
   <addaction name="menu_H"/>
  </widget>
  <action name="action_Q">
   <property name="text">
    <string>終了(&amp;Q)</string>
   </property>
   <property name="toolTip">
    <string>終了(Q)</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="action_Z">
   <property name="text">
    <string>ぐるぐるモード(&amp;Z)</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>修正モード(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="action_M">
   <property name="text">
    <string>形状記憶(&amp;M)</string>
   </property>
   <property name="toolTip">
    <string>形状記憶(M)</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>形状復帰(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>形状復帰(R)</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>展開図出力(&amp;6)</string>
   </property>
   <property name="toolTip">
    <string>展開図出力(6)</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>スナップショット出力(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>スナップショット出力(S)</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>設定(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>設定(C)</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>README(&amp;H)</string>
   </property>
   <property name="toolTip">
    <string>README(H)</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>FullereneViewerについて(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Q</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
